### PR TITLE
Chap10 edits

### PR DESCRIPTION
**Pull request by CarlyLanglois**

### CODE REVIEW

File: src/chapters/functions/composing-functions.rst [hunk 1]
```diff
@@ -161,7 +135,7 @@ This function is still short, which is good. However, it does two separate
 jobs---it reverses a string *and* decides if that string is a palindrome.
 
 Making a palindrome checker with one function vs. two might not seem like a big
-deal, but what if we need to reverse a string for some other reason? We cannot
+deal now. But what if we need to reverse a string for some other reason? We cannot
```
Comment (jimflores5) [inline src/chapters/functions/composing-functions.rst:138]: I know _technically_ it's OK to start a sentence with "But", but Mrs. Smith (my old English teacher) is screaming in my head right now.  Can we re-work this?

File: src/chapters/functions/why-create-functions.rst [hunk 2]
```diff
@@ -54,7 +54,7 @@ create a custom-made replacement part, and then weld it into place.
 
 The repair complexity goes way down when a car is made up of lots of small
 parts, each of which can be tested and replaced on its own. The same thing
-happens with code. Small, clear chunks of code are easier to manage.
+happens with code. Small, uncomplicated chunks of code are easier to manage.
```
Comment (jimflores5) [inline src/chapters/functions/why-create-functions.rst:57]: How about "simple" or "simpler" instead of "uncomplicated"?

**Review by jimflores5 — CHANGES_REQUESTED** [review]
Carly, most of my comments are just simple, small changes.  There are some questions that might need another round of back-and-forth, though.

File: src/chapters/functions/composing-functions.rst [hunk 3]
```diff
@@ -31,46 +31,22 @@ Palindrome Checker
...
-   def reverse_string(a_string):
-      rev_string = ''
-
-      for char in a_string:
-         rev_string = char + rev_string
-
-      return rev_string
-
-This is the same algorithm that we used in the loops chapter to
-:ref:`reverse a string <reverse-string>`.
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/functions/composing-functions.rst:71]: the other example on the page does link back to where they have seen it before and provides a breakdown of the code. I think adding another example of how to reverse a string is indeed distracting. as the reader I had a hard time figuring out why it was provided

File: src/chapters/functions/composing-functions.rst (hunk 1 above)
Comment (CarlyLanglois) [reply src/chapters/functions/composing-functions.rst:138]: We have plenty of sentences in the book that begin with conjunctions - this is just another one!

File: src/chapters/functions/calling-functions.rst [hunk 4]
```diff
@@ -80,7 +80,7 @@ to do those jobs for us with the data we supply.
    Here is an import vocabulary term!
    
    **Arguments** refer to the data values we send to a function. Some functions
-   do not require arguments inside the ``()``, but most will.
+   do not require arguments inside the parentheses, ``()``.
```
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/functions/calling-functions.rst:83]: OK.

File: src/chapters/functions/composing-functions.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/functions/composing-functions.rst:71]: Gotcha.  We'll stick with the one link then (to the split/join example).

File: src/chapters/functions/composing-functions.rst (hunk 1 above)
Comment (jimflores5) [reply src/chapters/functions/composing-functions.rst:138]: Ha!  This made me check.  3 total for "And", "Or", and "But" combined.  I guess we'll add a fourth, but it's killing me...

**Review by jimflores5 — APPROVED** [review]
All clear!

**Pull request merged**